src/plone/restapi/deserializer/dxcontent.py

```diff
@@ -25,6 +26,9 @@
 from .mixins import OrderingMixin
 
 
+MARKER = object()
```

Comment:
@thet could you elaborate why you define this outside of the class?

Reply:
No specific reason - only, that this feels more efficient. Initialized once, reusable anywhere.
Do you prefer to have it within the scope of the block where it is used? I'm fine with that.

Reply:
@thet I'd prefer readability over premature optimization. :) Therefore I'd put it next to where it is used.